### PR TITLE
Styling up the links

### PR DESCRIPTION
**Pull request by rdjpalmer**
## What

Style links to match the default styles we want, after seeing static page designs and through discussion with @scott-riley.

Also includes a border override modifier class

## Updated preview

screen shot 2016-01-27 at 09 18 20

### CODE REVIEW

File: globals/typography.css [hunk 1]
```diff
@@ -42,3 +42,38 @@
...
+
+.link {
+  text-decoration: none;
+  color: var(--color-primary);
+  font-style: italic;
+  background-image: linear-gradient(to top,
+    transparent 11%,
+    color(var(--color-primary) a(30%)) 11%,
+    color(var(--color-primary) a(30%)) 16%,
+    transparent 16%
```
Comment (iest) [inline globals/typography.css]: dafuq

**Comment by iest** [comment]
`text-decoration: underline` not good enough for you?

File: globals/typography.css (hunk 1 above)
Comment (rdjpalmer) [inline globals/typography.css]: https://medium.com/designing-medium/crafting-link-underlines-on-medium-7c03a9274f9 / http://codepen.io/timbrown/pen/kuDxK

As per discussion with @scott-riley

Comment (iest) [inline globals/typography.css]: You lose out in safari...

screenshot 2016-01-26 14 56 28

**Comment by scott-riley** [comment]
Link underlines aren’t the same colour as links because it makes them unreadable as fuck, but it’s mostly an aesthetic choice so go with whatever you think is appropriate m8

**Comment by rdjpalmer** [comment]
+1 on readability of the links in this PR. If the codes good, I'd like to ship unless theres any major reason not to @iest

**Comment by iest** [comment]
screenshot 2016-01-26 15 10 27

**Comment by iest** [comment]
On smaller font sizes the line will disappear on the background-image hack.

Also we lose out on Safari's (aka iOS's) pretty underlines.

What's the upside?

**Comment by iest** [comment]
:ship:

**Pull request merged**